index.html

```diff
@@ -12,8 +12,10 @@
 <body>
 	<!-- APP CONTENT -->
 	<div class='container'>
-		<h1 class='col-md-12'>Robot Battledome</h1>
-		<div class='mainContent'></div>
+		<div class='row'>
+			<h1 class='col-md-12'>Robot Battledome</h1>
+			<div class='mainContent'></div>
+		</div>
```

Comment:
You may want to pull this closing `div` up a line, as it is, all your JS-added rows will be inside this row, which may lead to odd behavior.